data_structures/queue_implementation.cpp

```diff
@@ -0,0 +1,109 @@
+/**
+ * @author [saqlain] (https://github.com/Mysterious786)
+ * @file
+ *
+ * Implementation of a Queue using Single Stacks.
+ *
+ */
+
+#include <iostream>
+#include <stack>
+#include <cstdlib>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c++
#include <iostream>    /// for IO operations
#include <cassert>    /// for assert
```

Reply:
Ok